readme.md

```diff
@@ -644,6 +644,7 @@
 - [Security Card Games](https://github.com/Karneades/awesome-security-card-games#readme) - Train your skills and discuss various security topics.
 - [Suricata](https://github.com/satta/awesome-suricata#readme) - Intrusion detection/prevention system and network security monitoring engine.
 - [Prompt Injection](https://github.com/FonduAI/awesome-prompt-injection#readme) - A type of vulnerability that specifically targets machine learning models.
+- [Detection Engineering](https://github.com/infosecB/awesome-detection-engineering#readme) - A curated list of tools and resources for Threat Detection Engineers.
```

Comment:
> Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb. It should not contain the name of the list.

Reply:
Updated description in [bfcc6e1](https://github.com/sindresorhus/awesome/pull/2829/commits/bfcc6e12343e99192f6893426882d4a3c8f2bc2d) per your feedback. Thank you.